charts/substra-backend/templates/statefulset-worker.yaml

```diff
@@ -128,6 +128,16 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
```

Comment:
Can you please explain why this is needed/what it is for? Is it to match the labelSelector on `requiredDuringSchedulingIgnoredDuringExecution`? I think it might be worth adding on comment on that part.

Reply:
It's used for the [dependent variables](https://kubernetes.io/docs/tasks/inject-data-application/define-interdependent-environment-variables/), which is used for the default value of `.Values.worker.computePod.affinity`  but could also be used in the value file. there is a comment in the value file [about its use](https://github.com/Substra/substra-backend/pull/935/files/1388241a722b994d9022bc06e25abbab8fd5e6c5#diff-f506fbc1c868db1eb769f2236b9a9a2c2794484d8bb4a1723fd598016ac8c033R286), and could be used to define other affinity policies refering to the pod name. 

Which kind of comment would you like to see? a link to the Kubernetes documentation and a text explaining that we use dependent variables?

Reply:
It is more about explaining why it is needed in this context (and the comment was for `POD_NAME` - not `COMPUTE_POD_*`) but nvm